src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -275,4 +376,4 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 	};
 
 	$( obj.ready );
-} )( jQuery, tribe.tickets.commerce.gateway.stripe, Stripe, ky );
\ No newline at end of file
+})( jQuery, tribe.tickets.commerce, Stripe, ky );
```

Comment:
Same comment here about switching to `tribe.tickets.commerce`

Reply:
@juanfra we're moving to `tribe.tickets.commerce` because there is a new key in `tribe.tickets.commerce.billing` that we need. The other option would be to add another key here and use `obj` for `tribe.tickets.commerce.gateway.stripe.checkout` and `billing` for `tribe.tickets.commerce.billing` both can work,